app/views/user_mailer/send_key.html.erb

```diff
@@ -3,19 +3,78 @@
   <head>
     <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
   </head>
-  <body>
-    <h1>Dear user</h1>
-    <p>
-      We appreciate your preference.
+      <body style="margin: 0; padding: 0;">
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td style="padding: 20px 0 30px 0;">
 
-      Here is your access key. Enjoy and protect yourself.
+  <table align="center" border="0" cellpadding="0" cellspacing="0" width="600" style="border-collapse: collapse; border: 1px solid #cccccc;">
```

Comment:
CSS in HTML?? I think it's not a good practice, be careful

Reply:
It wasn't under my control bro